zips/zip-0227.rst

```diff
@@ -244,6 +244,14 @@ An Issue Note is a tuple $(\mathsf{d}, \mathsf{pk_d}, \mathsf{v}, \mathsf{AssetB
 
 The complete encoding of these fields into an ``IssueNote`` is defined in ZIP 230 [#zip-0230-issue-note]_.
 
+Let $\mathsf{Note^{Issue}}$ be the type of an Issue Note, i.e.
+
+.. math:: \mathsf{Note^{Issue}} := \mathbb{B}^{[\ell_{\mathsf{d}}]} \times \mathsf{KA}^{\mathsf{Orchard}}.\mathsf{Public} \times \{0 .. \mathsf{MAX\_ISSUE}\} \times \mathbb{P}^* \times \mathbb{F}_{q_{\mathbb{P}}} \times \mathbb{B}^{[\mathbb{Y}^{32}]}. 
```

Comment:
Why here we have `Max_issue` and in zip226 we have 0-2^l.
Probable we need the same type for both.

Reply:
same comment for the domain for asset base.

Reply:
We can continue to use $\{0 .. 2^{\ell} - 1\}$ for both, and use MAX_ISSUE only in the consensus rule check.

Reply:
Updated